.github/workflows/base-cleanup-appconfig-and-lambda-layer-versions.yml

```diff
@@ -1,4 +1,4 @@
-name: 'Cleanup Untracked App Config Profiles and Lambda Layer Versions'
+name: 'Z-BASE: Cleanup (App Config Profiles & Lambda Layer Versions)'
```

Comment:
Maybe a more generic name so that this can be used for other cleanup if needed in the future. 
"Z-BASE: Workspace Cleanup" ?